### PR TITLE
Add missing fields to read_lua_file for BatchNorm and Linear layers.

### PR DESCRIPTION
**Pull request by bamos**
Without these, I'm getting errors when loading models from Lua Torch.

### CODE REVIEW

**Comment by apaszke** [comment]
Could you add entries like [these](https://github.com/pytorch/pytorch/blob/master/torch/utils/serialization/read_lua_file.py#L354) instead of changing the modules? First argument is module name, others are attributes that will be set to `None` if they're not present. Thanks!

**Comment by bamos** [comment]
Sure, updated.

**Comment by apaszke** [comment]
Thanks!
